src/include/duckdb/common/types/timestamp.hpp

```diff
@@ -189,6 +191,7 @@ class Timestamp {
 	DUCKDB_API static int64_t GetEpochMicroSeconds(timestamp_t timestamp);
 	//! Convert a timestamp to epoch (in nanoseconds)
 	DUCKDB_API static int64_t GetEpochNanoSeconds(timestamp_t timestamp);
+	DUCKDB_API static int64_t GetEpochNanoSeconds(timestamp_ns_t timestamp);
```

Comment:
This is the same as directly accessing `value`, no? I'm unsure if we need it, but I'm also okay with adding it.

Reply:
> This is the same as directly accessing value, no?

Yes. But so is [`GetEpochMicroSeconds(timestamp_t timestamp)`](https://github.com/andreimatei/duckdb/blob/da42e2d9fcebbc302547dda516b546bda6577870/src/common/types/timestamp.cpp#L430-L432) -- and that function exists.
So, if it's up to me, I'd leave it. But I don't know this code at all, so...